src/TimeModeling/Modeling/utils.jl

```diff
@@ -1,10 +1,12 @@
 
-
-function judipmap(func, iter::UnitRange; )
-    # Ignore map and workers if single source
-    length(iter) == 1 && (return [func(iter[1])])
-    # Switch to asyncmap if serial julia
-    length(default_worker_pool()) < 2 && (return asyncmap(func, iter))
-    # Standard pmap if parallel and multiple sources
-    return pmap(func, iter)
+function judipmap(func, iter...; on_error = nothing)
+    # judipmap for multiple indices
+    # Ignore map and workers if single task
+    if prod(length(iter[i]) for i = 1:length(iter)) == 1
```

Comment:
Why is this still here?

Reply:
This is an extension for `judipmap` anyway to support more input indices and `on_error` option?

Reply:
It's not used or needed anywhere so no
`on_error` yes ok.